timeseries/src/autogluon/timeseries/evaluator.py

```diff
@@ -219,6 +240,9 @@ def save_past_metrics(self, data_past: TimeSeriesDataFrame):
             y_past=data_past[self.target_column], seasonal_period=seasonal_period
         )
 
+        if self.eval_metric == "RMSSE":
```

Comment:
Should we add a similar check above and only compute the `_past_naive_error` in case `eval_metric` is MASE?

Reply:
This would also require changing the test https://github.com/autogluon/autogluon/blob/0a00e0652321e67e8b4f9e26eac8bdef21cd4628/timeseries/tests/unittests/test_evaluator.py#L196

Maybe add a similar test that ensures that squared error is computed for RMSSE?

Reply:
I did a very small updated on the test, but I am not sure if I understand enough what needs to be updated here. 
What I did update is `score_with_saved_past_metrics`, given that certain attributed are stored only if certain `eval_metrics` are used.